packages/pharos/src/components/popover/PharosPopover.react.stories.jsx

```diff
@@ -0,0 +1,121 @@
+    docs: {
+      page: configureDocsPage('popover'),
+    },
+  },
+};
+
+export const Base = {
+  render: () => (
+    <div>
+      <PharosButton id="my-button" data-popover-id="my-popover" icon-right="chevron-down">
+        Click Me
+      </PharosButton>
+      <PharosPopover id="my-popover">
+        <div style="padding: 1rem">Some very simple contents</div>
+      </PharosPopover>
+    </div>
+  ),
+};
+
+export const Events = {
+  render: () => (
+    <div>
+      <PharosButton id="my-button" data-popover-id="my-popover" icon-right="chevron-down">
+        Click Me
+      </PharosButton>
+      <PharosPopover id="my-popover">
+        <div style="padding: 1rem; display: flex; flex-direction: column; gap: 1rem;">
+          <div style="padding: 1rem;">Some very simple contents</div>
+          <PharosButton
+            onClick={() => {
+              const menu = document.querySelector('storybook-pharos-popover');
+              menu.open = false;
+            }}
+          >
+            Close
+          </PharosButton>
+        </div>
+      </PharosPopover>
+    </div>
+  ),
+};
+
+export const OnBackground = {
+  render: () => (
+    <div>
+      <PharosButton id="my-button" data-popover-id="my-popover" icon-right="chevron-down">
+        Click Me
+      </PharosButton>
+      <PharosPopover id="my-popover" is-on-background>
+        <div style="background: #444444; color: white; padding: 1rem; display: flex; flex-direction: column; gap: 1rem;">
+          <div style="padding: 1rem">Some very simple contents</div>
+          <PharosButton
+            onClick={() => {
+              const menu = document.querySelector('storybook-pharos-popover');
+              menu.open = false;
+            }}
+          >
+            Close
+          </PharosButton>
+        </div>
+      </PharosPopover>
+    </div>
+  ),
+  parameters: {
+    backgrounds: { default: 'dark' },
+  },
+};
+
+export const LargeContents = {
+  render: () => (
+    <div>
+      <PharosButton id="my-button" data-popover-id="my-popover" icon-right="chevron-down">
+        Click Me
+      </PharosButton>
+      <PharosPopover id="my-popover">
+        <div style="padding: 1rem; display: flex; flex-direction: column; gap: 1rem;">
+          <div style="height: 200px; overflow: auto; border: 1px solid black; padding: 1rem;">
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
+            <div>Some really cool stuff</div>
```

Comment:
We typically put some slightly higher fidelity content into these for demonstrative purposes. Maybe @lorilundy711 can make a suggestion here that is in keeping with other components' content?

Reply:
I converted this to a paragraph of lorem ipsum text as a start but can do whatever if we want different content in there